plugins/git-build.js

```diff
@@ -35,7 +35,7 @@ function buildConsumer(config, cimpler, repoPath) {
             BUILD_STATUS: build.status,
             BUILD_QUEUED_AT: build.queuedAt,
          },
-         timeout: config.timeout || 0,
+         timeout: process.env.buildTimeout || config.timeout || 0,
```

Comment:
Hmmm...  I think this won't help cause `process.env` here is the cimpler process. We alread have a way to specify a timeout for the plugin, what we want is a way to specify the timeout for a particular build.

Perhaps provide a `--timeout` argument to the `build` command and have it sent as `build.timeout`

Reply:
Is that what you were thinking? https://github.com/danielbeardsley/cimpler/pull/104/commits/402110fdd07d4642cb8aef2af6259e1f06f5960c